Classes/BITCrashCXXExceptionHandler.mm

```diff
@@ -58,7 +58,7 @@
 
 @implementation BITCrashUncaughtCXXExceptionHandlerManager
 
-extern "C" void LIBCXXABI_NORETURN __cxa_throw(void *exception_object, std::type_info *tinfo, void (*dest)(void *))
```

Comment:
@diederich Hm, interesting. Tempted to just use `_LIBCXXABI_NORETURN` again, looks a bit nicer. Thanks for finding the change!